pulp_rpm/tests/functional/api/test_fips_workflow.py

```diff
@@ -0,0 +1,965 @@
+# coding=utf-8
```

Comment:
nitpick: can these utf8 lines be removed. Python3 defaults to utf8 now.

Reply:
There's more of those, including in the template.  Let's do it in a different PR (/ set of PRs).

Reply:
Sounds fine, but who will make that PR and when?

Reply:
pulp_rpm, pulp_file, and pulp_container tests (at least) are full of these. I've opened issues for each project, and will get at least pulp_rpm addressed in this sprint.

Reply:
Also can one be opened for the plugin_template too please?

Reply:
See https://pulp.plan.io/issues/8223, 8224, 8225, 8226 for RPM, File, Container, and Pulp/Plugin-Template. I think we're good to go.

Reply:
Thank you!